### PR TITLE
Bl 1115 fe make case details search bar functional

### PR DESCRIPTION
**Pull request by Kobramall**
loom: 
 https://www.loom.com/share/229ce96a9c1c4b308c198edb5a50ba33

I replace the place holder array with the notes to it could filter through it when using the search bar.
I also deleted the  place holder array because it was no longer being used

### CODE REVIEW

File: src/components/common/CaseDetails/CaseDetails.js [hunk 1]
```diff
@@ -30,12 +31,23 @@ function CaseDetails() {
...
+  const clientId = 1;
+
+  //fetching notes from the backend with axios
+  useEffect(() => {
+    axios
+      .get(`http://localhost:8000/api/clients/${clientId}/notes`)
+      .then(res => setNotes(res.data))
+      .catch(err => console.log(err));
+  }, []);
+
```
Comment (JordonP123) [inline src/components/common/CaseDetails/CaseDetails.js:50]: I think we are putting all axios calls inside of the api file!

Comment (JordonP123) [reply src/components/common/CaseDetails/CaseDetails.js:50]: If you need help with that just let me know!

File: src/components/common/CaseDetails/CaseDetails.js [hunk 2]
```diff
@@ -79,17 +79,22 @@ function CaseDetails() {
...
-      delete_at: null,
       created_by: '1',
     };
 
     //Post the note to the backend
-    setNotes([...notes, savedNote]);
+    axios
+      .post(`http://localhost:8000/api/clients/${clientId}/notes`, savedNote)
+      .then(res => console.log(res.data))
+      .catch(err => console.error(err.message));
```
Comment (JordonP123) [inline src/components/common/CaseDetails/CaseDetails.js:81]: Not sure if we are addding the posts to the api file, maybe double check with Jake on that one.  But looks solid!

File: src/components/common/CaseDetails/CaseDetails.js [hunk 3]
```diff
@@ -210,10 +206,18 @@ function CaseDetails() {
         <div className="Rightside__PreviousNotes">
           {filterSearch.length > 1
             ? filteredResults.map(item => {
+                let date = new Date(item.created_at);
```
Comment (JordonP123) [inline src/components/common/CaseDetails/CaseDetails.js:209]: There is a method to make this a one liner, but no need lol.  Looks solid!

File: src/components/common/CaseDetails/CaseDetails.js [hunk 4]
```diff
@@ -28,9 +29,25 @@ function CaseDetails() {
   const [filterSearch, setFilterSearch] = useState('');
   const [filteredResults, setFilteredResults] = useState([]);
 
+  //State slice for notes
+  const [notes, setNotes] = useState([
+    { message: 'place holder note', date: '11/9/2022' },
```
Comment (JordonP123) [inline src/components/common/CaseDetails/CaseDetails.js:34]: You might want this to start as an empty object, now that it is functional

**Review by JordonP123 — COMMENTED** [review]
Overall Solid work!  I would just double check with Jake about the api calls if he wants posts/get to be in the api folder

File: src/components/common/CaseDetails/CaseDetails.js [hunk 5]
```diff
@@ -210,10 +206,18 @@ function CaseDetails() {
         <div className="Rightside__PreviousNotes">
           {filterSearch.length > 1
             ? filteredResults.map(item => {
+                let date = new Date(item.created_at);
+                let month = date.getMonth();
```
Comment (ramib5734) [inline src/components/common/CaseDetails/CaseDetails.js:210]: Good use of these methods!

**Pull request merged**